libs/experimental/langchain_experimental/data_anonymizer/presidio.py

```diff
@@ -44,12 +46,23 @@
     from presidio_analyzer import EntityRecognizer, RecognizerResult
     from presidio_anonymizer.entities import EngineResult
 
+DEFAULT_LANGUAGES_CONFIG = {
+    "nlp_engine_name": "spacy",
+    "models": [
+        {"lang_code": "en", "model_name": "en_core_web_lg"},
+        # {"lang_code": "de", "model_name": "de_core_news_md"},
```

Comment:
Can you add a link to where one can find model names? It probably would be more helpful for people.

Reply:
I saw it is mentioned in the notebook, but I think it is nice to get information directly from code editor, without jumping to docs.